src/_imagingcms.c

```diff
@@ -390,7 +389,7 @@ _buildTransform(
 
     Py_END_ALLOW_THREADS
```

Comment:
```suggestion
    Py_END_ALLOW_THREADS;
```

Reply:
At the time of #4770 this caused a compiler error on Windows; not sure if it is supported now.